test/unittests/state_mpt_test.cpp

```diff
@@ -164,6 +164,9 @@ TEST(state_mpt, trie_topologies)
         const char* hash_hex;
     };
 
+    // The test cases are cross-checked with go-ethereum implementation.
+    // https://github.com/ethereum/go-ethereum/blob/master/trie/stacktrie_test.go#L35
```

Comment:
permalink
```suggestion
    // https://github.com/ethereum/go-ethereum/blob/7dea9c10cdb42e8c9f71b8b324cbe9222ab105cf/trie/stacktrie_test.go#L35
```

Reply:
It could be, but I'm not sure. Maybe some other test are going to be added there later. On the other hand fixed commit hash will be information when these were in sync...